reactDashboard/src/components/RouteMapRules/RegionalRouteMapRules.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import "./RouteMapRules.css";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+
+const RegionalRouteMapRules = () => {
```

Comment:
Added new component using the text and links laid out in [HIT-194](https://sta2020.atlassian.net/browse/HIT-194)